src/main/java/com/coveo/configuration/parameterstore/ParameterStorePropertySourceConfigurationProperties.java

```diff
@@ -19,8 +20,9 @@
                                                                        "signingRegion");
     public static final String MULTI_REGION_SSM_CLIENT_REGIONS = joinWithDot(SOURCE_PREFIX,
                                                                              "multiRegion",
-                                                                             "ssmClient",
+                                                                             SSM_CLIENT,
                                                                              "regions");
+    public static final String MAX_ERROR_RETRY = joinWithDot(SOURCE_PREFIX, SSM_CLIENT, "maxErrorRetry");
```

Comment:
`maxErrorRetries` like in the getter?

Reply:
the aws parameter is maxErrorRetry, so I kept the same naming